interfaces/certificate_transfer/v0/README.md

```diff
@@ -25,15 +26,16 @@ Both the Requirer and the Provider need to adhere to criteria to be considered c
 
 ### Requirer
 
-- Is expected to use the provided certificate and/or CA certificate and/or chain to authenticate communications.
+- Is expected to send its highest supported version number and use the provided certificate and/or CA certificate and/or chain to authenticate communications.
```

Comment:
I'm not sure about the "highest supported version" phrasing. This version number is the version of the interface definition being used.

I don't think it's necessarily true that it's the highest supported version. For instance, a lib could support v0, and v1, and expose that version through an instantiation parameter, allowing users to explicitely decide the version they want to use.

Reply:
For this interface, if we keep to a single version number, I think it should be the highest supported version. I do not think we want to start exposing configuration item for this, as the goal is to help smoothly move things along to the newer version.

This is also the documentation for v0, I could reword this to say that it's expected to send version 0, as there is no other option when using v0.

Reply:
I pushed a small change for this.